<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SelectAddress</class>
 <widget class="QDialog" name="SelectAddress">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>406</width>
    <height>300</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>14</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>Регион:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="cmbRegion">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="editable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Город:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="cmbSity">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="editable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Улица:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="cmbStreet">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="editable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Дом:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="cmbHaus">
       <property name="editable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Корп.</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="leCorp">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>Кв.</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="leKvartira"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>SelectAddress</receiver>
   <slot>setAddress()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>257</x>
     <y>290</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>SelectAddress</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cmbRegion</sender>
   <signal>activated(QString)</signal>
   <receiver>SelectAddress</receiver>
   <slot>getSity()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>337</x>
     <y>35</y>
    </hint>
    <hint type="destinationlabel">
     <x>395</x>
     <y>7</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cmbSity</sender>
   <signal>activated(QString)</signal>
   <receiver>SelectAddress</receiver>
   <slot>getStreet()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>373</x>
     <y>106</y>
    </hint>
    <hint type="destinationlabel">
     <x>393</x>
     <y>106</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cmbStreet</sender>
   <signal>activated(QString)</signal>
   <receiver>SelectAddress</receiver>
   <slot>getHaus()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>340</x>
     <y>165</y>
    </hint>
    <hint type="destinationlabel">
     <x>388</x>
     <y>181</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>getSity()</slot>
  <slot>getStreet()</slot>
  <slot>getHaus()</slot>
  <slot>setAddress()</slot>
 </slots>
</ui>
